components/Slider/examples/SliderCollectionViewController.m

```diff
@@ -82,12 +82,14 @@ - (void)applyModel:(MDCSliderModel *)model withColorScheme:(MDCSemanticColorSche
 @implementation MDCSliderExampleCollectionViewCell {
   UILabel *_label;
   MDCSlider *_slider;
+  MDCTypographyScheme *_typographyScheme;
 }
 
 - (instancetype)initWithFrame:(CGRect)frame {
   if (self = [super initWithFrame:frame]) {
+    _typographyScheme = [[MDCTypographyScheme alloc] init];
     _label = [[UILabel alloc] init];
-    _label.font = [MDCTypography body1Font];
+    _label.font = _typographyScheme.subtitle2;
```

Comment:
Looks like this should be inherited from the example. Can you please add a settable property? Or maybe just have the VC set the label.font directly?

Reply:
Done